aemrules.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Adds rules for AEM Java development
 homepageUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube
-archivedVersions=0.8,0.9,0.10,0.11
-publicVersions=1.0
+archivedVersions=0.8,0.9,0.10,0.11,1.0
+publicVersions=1.1
 
 defaults.mavenGroupId=com.cognifide.aemrules
 defaults.mavenArtifactId=sonar-aemrules-plugin
 
+1.1.description=Java Plugin 6.2 support
+1.1.sqVersions=[7.9,LATEST]
```

Comment:
Now that I'm actually _reading_ your release notes, is this the correct range minimum? Your notes list 7.9.2 as the SQ min. (versus 7.9[.0]) and v6.0 of the Java analyzer. SonarQube 8.1 shipped with a 5.x version & 8.2 shipped with v6.1

Reply:
Hi,
I fixed release notes, there should be 7.9.0. Issue about versions of Java analyser is that SQ forces to update plugin on server restart. So, if I install AEM Rules 1.0 on SQ 7.9.x, after server restart, I’ll have AEM Rules 1.0 and Java analyser 6.2, this configuration fails while scanning code.  That’s why I want to AEM Rules 1.1 to support Java analyser 6.x. Of course, if someone want to install AEM Rules manually via jar file on SQ with 5.x Java analyser should use AEM 1.0.
I don’t exactly know if this is right solution for this issue.

Thanks for your analysis.

Reply:
We can't explicitly manage dependency versions here. That's handled based on the dependencies you declare. So I think we're good enough here.